### ISSUE TITLE
Lighthouse Chrome Extension hangs indefinitely and silently on "Retrieving: ResponseCompression"

### ISSUE THREAD

**Issue by kwidholm-tm** [GoogleChrome/lighthouse#2604]
When running a chrome extension lighthouse report against a site we are building, we have noticed that the extension will hang indefinitely during "Retrieving: ResponseCompression". 

![screen shot 2017-06-27 at 13 10 45](https://user-images.githubusercontent.com/12515746/27604454-4e2245dc-5b47-11e7-88e0-148b0734a9a7.png)

No failure message is presented, and nothing appears in the console to indicate why the extension stops at this juncture. This does not happen for all sites, but there is no output capturing the failure or helping indicate the reason for the issue.

Further, when running lighthouse from the command line, we do not experience this issue. The tests complete fine, so this seems to be an issue specific to the chrome extension. 

Is there a way to run the chrome extension in some kind of verbose or debug mode so that I can better document the issue?

Thanks.

**Comment by patrickhulce** [comment]
@kwidholm-tm are you inspecting the extension's background script when you say "nothing appears in the console"? If not you can inspect the background script of an extension by going to `chrome://inspect/#extensions`.

**Comment by kwidholm-tm** [comment]
@patrickhulce Thank you! I didn't know about inspecting extensions, and it's very helpful.

However, inspection did not yield any further insight. I would have inspected, at least a timeout error or invalid response or dropped connection, but lighthouse hangs indefinitely at this point, and, even after an hour or two, no further output is recorded and progress does not complete:

![screen shot 2017-06-27 at 17 08 47](https://user-images.githubusercontent.com/12515746/27610249-97286a86-5b5b-11e7-9eca-d74700748dc2.png)

```
lighthouse-background.js:44854 status Retrieving: PasswordInputsWithPreventedPaste +3ms
lighthouse-background.js:44854 status Retrieving: ResponseCompression +4ms
```

**Comment by patrickhulce** [comment]
Pretty nifty right? :)

Hm, that is strange then, do you have this site on a publicly accessible URL somewhere we can take a look?

**Comment by kwidholm-tm** [comment]
@patrickhulce No the site is not publicly accessible. You have to be on our dev network to load it. I'll see if I can set up a temporary proxy to it and will post back here once one is set up.

In any case, this issue is not critical in that we can generate the reports from the command line just fine; it was just _nice_ to be able to use the extension.

**Comment by kwidholm-tm** [comment]
@patrickhulce It looks like I would have to whitelist a client IP temporarily in order to get you through to it. I tried using ngrok but due to application data fetching and how it is set up, that's not going to work. If you feel like looking into this and have a stable client IP I can see about temporarily allowing access.

**Comment by patrickhulce** [comment]
It's going to be tough to fix without a consistent repro on our side, if you can find another URL that also hangs or can create a reduced problem case that demonstrates the behavior let us know!

My gut says that it's an issue with `pako` which is the browser replacement we use for gzipping the content. You could start by trying to compress your resources with pako and seeing if those hang?

**Comment by wardpeet** [comment]
@kwidholm-tm could you copy the whole log maybe it's another command in the chain that messes it up.

Also does the lighthouse cli works?

**Comment by kwidholm-tm** [comment]
@patrickhulce @wardpeet  Good news and bad news. The bad news is that I can't generate a meaningful full log anymore because now the lighthouse extension is now able to complete its run. The good news is that the extension is now able to complete its run and doesn't hang.

Something in our latest version of the app seems to have unblocked the extension from hanging, and I am upset that I can't see what the issue was or why it was blocking. The cli was working all along, and it was just the extension that was hanging on `Retrieving: ResponseCompression`

I hate to close things out without any lessons learned, but let's close this. If I run into it again, I'll reopen and this time with a full inspection log.

Thanks for peeking at this oddity with me.

**Comment by patrickhulce** [comment]
no worries @kwidholm-tm thanks for trying! I'm definitely curious too, so stop back again if you uncover more details :)

**Comment by WillsB3** [comment]
I'm also seeing this error while testing one of our sites. Things I've tried:

- Using a fresh Chrome profile with just the Lighthouse extension installed
- Inspecting the extension - Nothing of note in the console, just hung on `status Retrieving: ResponseCompression +4ms`

**Comment by WillsB3** [comment]
Ha.. Managed to get it working. Must have been PEBCAK...

For future people who have this issue:

- Are you sure you are using the right URL?
- Are you sure the server at the URL your entered it running/responding as expected?
- If you are using `chrome-debug` to login to your site before running Lighthouse, are you sure you entered the correct port when specifying the `--port` argument in the call to `lighthouse`.

**Comment by markkelly1982** [comment]
Hi all,

I have the same issue. The app is just a test app and is accessible at http://mk-design.me/angular/recipes-app/

The console log returns the following

`lighthouse-background.js:45267 Lighthouse:warn The URL provided should be on HTTPS +0ms
lighthouse-background.js:45267 Lighthouse:warn Performance stats will be skewed redirecting from HTTP to HTTPS. +3ms
lighthouse-background.js:45267 <= Page.frameStartedLoading +103ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Page.frameNavigated +7ms Object {frame: Object}
lighthouse-background.js:45267 <= Page.loadEventFired +14ms Object {timestamp: 253237.965782}
lighthouse-background.js:45267 <= Page.frameStoppedLoading +8ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Page.domContentEventFired +4ms Object {timestamp: 253237.971609}
lighthouse-background.js:45267 <= Page.frameResized +14ms Object {}
lighthouse-background.js:45267 <= Page.frameResized +78ms Object {}
lighthouse-background.js:45267 status Initializing… +214ms
lighthouse-background.js:45267 <= ServiceWorker.workerRegistrationUpdated +4ms Object {registrations: Array(7)}
lighthouse-background.js:45267 <= ServiceWorker.workerVersionUpdated +4ms Object {versions: Array(7)}
lighthouse-background.js:45267 <= ServiceWorker.workerRegistrationUpdated +7ms Object {registrations: Array(7)}
lighthouse-background.js:45267 <= ServiceWorker.workerVersionUpdated +3ms Object {versions: Array(7)}
lighthouse-background.js:45267 <= Page.frameResized +10ms Object {}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +12ms Object {context: Object}
lighthouse-background.js:45267 <= Page.frameStartedLoading +51ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Runtime.executionContextDestroyed +3ms Object {executionContextId: 11}
lighthouse-background.js:45267 <= Runtime.executionContextsCleared +3ms Object {}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Page.frameNavigated +3ms Object {frame: Object}
lighthouse-background.js:45267 <= Page.loadEventFired +2ms Object {timestamp: 253238.401543}
lighthouse-background.js:45267 <= Page.frameStoppedLoading +8ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Page.domContentEventFired +4ms Object {timestamp: 253238.406642}
lighthouse-background.js:45267 status Loading page & waiting for onload +323ms URL, Viewport, ViewportDimensions, ThemeColor, Manifest, ChromeConsoleMessages, ImageUsage, Accessibility, EventListeners, AnchorsWithNoRelNoopener, AppCacheManifest, DOMStats, OptimizedImages, PasswordInputsWithPreventedPaste, ResponseCompression, TagsBlockingFirstPaint, WebSQL
lighthouse-background.js:45267 method <= browser ERR:error Tracing.end +59ms 
lighthouse-background.js:45267 <= Page.frameStartedLoading +19ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.responseReceived +750ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.18", response: Object, timestamp: 253239.590599…}
lighthouse-background.js:45267 <= Network.dataReceived +28ms Object {dataLength: 809, encodedDataLength: 0, requestId: "4256.18", timestamp: 253239.591992}
lighthouse-background.js:45267 <= Runtime.executionContextDestroyed +4ms Object {executionContextId: 12}
lighthouse-background.js:45267 <= Runtime.executionContextsCleared +2ms Object {}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Page.frameNavigated +4ms Object {frame: Object}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 0, encodedDataLength: 809, requestId: "4256.18", timestamp: 253239.603538}
lighthouse-background.js:45267 <= Network.loadingFinished +6ms Object {encodedDataLength: 1127, requestId: "4256.18", timestamp: 253239.587282}
lighthouse-background.js:45267 <= Network.requestWillBeSent +845ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +2ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +2ms Object {context: Object}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Network.responseReceived +131ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.19", response: Object, timestamp: 253240.64272…}
lighthouse-background.js:45267 <= Network.dataReceived +8ms Object {dataLength: 3772, encodedDataLength: 0, requestId: "4256.19", timestamp: 253240.65155}
lighthouse-background.js:45267 <= Network.dataReceived +56ms Object {dataLength: 32768, encodedDataLength: 3772, requestId: "4256.19", timestamp: 253240.707888}
lighthouse-background.js:45267 <= Network.responseReceived +190ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.20", response: Object, timestamp: 253240.859044…}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.21", response: Object, timestamp: 253240.861039…}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 1122, encodedDataLength: 0, requestId: "4256.21", timestamp: 253240.862068}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.22", response: Object, timestamp: 253240.863072…}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 1122, encodedDataLength: 0, requestId: "4256.22", timestamp: 253240.864071}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.24", response: Object, timestamp: 253240.864893…}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 1122, encodedDataLength: 0, requestId: "4256.24", timestamp: 253240.865884}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.23", response: Object, timestamp: 253240.866894…}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 1120, encodedDataLength: 0, requestId: "4256.23", timestamp: 253240.867912}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 1535, encodedDataLength: 0, requestId: "4256.20", timestamp: 253240.873283}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 0, encodedDataLength: 1535, requestId: "4256.20", timestamp: 253240.874265}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 1871, requestId: "4256.20", timestamp: 253240.872441}
lighthouse-background.js:45267 <= Network.dataReceived +257ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.19", timestamp: 253241.18636}
lighthouse-background.js:45267 <= Network.dataReceived +525ms Object {dataLength: 24116, encodedDataLength: 1122, requestId: "4256.22", timestamp: 253241.523353}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 0, encodedDataLength: 24116, requestId: "4256.22", timestamp: 253241.524238}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 25576, requestId: "4256.22", timestamp: 253241.523161}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 31418, encodedDataLength: 1122, requestId: "4256.24", timestamp: 253241.658558}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 0, encodedDataLength: 31418, requestId: "4256.24", timestamp: 253241.659489}
lighthouse-background.js:45267 <= Network.loadingFinished +2ms Object {encodedDataLength: 32878, requestId: "4256.24", timestamp: 253241.658288}
lighthouse-background.js:45267 <= Network.dataReceived +4ms Object {dataLength: 32768, encodedDataLength: 1122, requestId: "4256.21", timestamp: 253241.682533}
lighthouse-background.js:45267 <= Network.dataReceived +7ms Object {dataLength: 32768, encodedDataLength: 1120, requestId: "4256.23", timestamp: 253241.734439}
lighthouse-background.js:45267 <= Network.dataReceived +8ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.19", timestamp: 253241.743565}
lighthouse-background.js:45267 <= Network.dataReceived +386ms Object {dataLength: 27735, encodedDataLength: 32768, requestId: "4256.21", timestamp: 253242.1278}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 0, encodedDataLength: 27735, requestId: "4256.21", timestamp: 253242.13005}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 61963, requestId: "4256.21", timestamp: 253242.127605}
lighthouse-background.js:45267 <= Network.dataReceived +65ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253242.198662}
lighthouse-background.js:45267 <= Network.dataReceived +24ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.19", timestamp: 253242.222681}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 911, encodedDataLength: 32768, requestId: "4256.19", timestamp: 253242.224548}
lighthouse-background.js:45267 <= Network.dataReceived +47ms Object {dataLength: 0, encodedDataLength: 911, requestId: "4256.19", timestamp: 253242.225985}
lighthouse-background.js:45267 <= Network.loadingFinished +2ms Object {encodedDataLength: 136079, requestId: "4256.19", timestamp: 253242.222728}
lighthouse-background.js:45267 <= Network.dataReceived +358ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253242.632595}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253242.634789}
lighthouse-background.js:45267 <= Network.dataReceived +105ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253242.73918}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253242.914605}
lighthouse-background.js:45267 <= Network.dataReceived +166ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.081317}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.255711}
lighthouse-background.js:45267 <= Network.dataReceived +176ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.430513}
lighthouse-background.js:45267 <= Network.dataReceived +174ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.605733}
lighthouse-background.js:45267 <= Network.dataReceived +176ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.781226}
lighthouse-background.js:45267 <= Network.dataReceived +174ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253243.955078}
lighthouse-background.js:45267 <= Network.dataReceived +167ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253244.122136}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253244.297479}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253244.472826}
lighthouse-background.js:45267 <= Network.dataReceived +151ms Object {dataLength: 28672, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253244.623781}
lighthouse-background.js:45267 <= Network.dataReceived +174ms Object {dataLength: 32768, encodedDataLength: 28672, requestId: "4256.23", timestamp: 253244.797461}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253244.97251}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253245.147726}
lighthouse-background.js:45267 <= Network.dataReceived +174ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253245.321759}
lighthouse-background.js:45267 <= Network.dataReceived +167ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253245.488965}
lighthouse-background.js:45267 <= Network.dataReceived +176ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253245.664502}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253245.839593}
lighthouse-background.js:45267 <= Network.dataReceived +173ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.013293}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.187998}
lighthouse-background.js:45267 <= Network.dataReceived +176ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.363561}
lighthouse-background.js:45267 <= Network.dataReceived +175ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.538129}
lighthouse-background.js:45267 <= Network.dataReceived +167ms Object {dataLength: 32768, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.704842}
lighthouse-background.js:45267 <= Network.dataReceived +135ms Object {dataLength: 24435, encodedDataLength: 32768, requestId: "4256.23", timestamp: 253246.839824}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 0, encodedDataLength: 24435, requestId: "4256.23", timestamp: 253246.842606}
lighthouse-background.js:45267 <= Network.loadingFinished +2ms Object {encodedDataLength: 939303, requestId: "4256.23", timestamp: 253246.839637}
lighthouse-background.js:45267 <= Network.requestWillBeSent +2s Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +3ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestWillBeSent +157ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Network.requestServedFromCache +3ms Object {requestId: "4256.33"}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.33", response: Object, timestamp: 253248.438132…}
lighthouse-background.js:45267 <= Network.dataReceived +4ms Object {dataLength: 192, encodedDataLength: 0, requestId: "4256.33", timestamp: 253248.438139}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 0, requestId: "4256.33", timestamp: 253248.438144}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +373ms Object {context: Object}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Page.domContentEventFired +2ms Object {timestamp: 253248.912273}
lighthouse-background.js:45267 <= Network.requestWillBeSent +33ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +2s Object {context: Object}
lighthouse-background.js:45267 <= Network.requestWillBeSent +2ms Object {documentURL: "http://mk-design.me/angular/recipes-app/", frameId: "4256.1", initiator: Object, loaderId: "4256.3", request: Object…}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +3ms Object {context: Object}
lighthouse-background.js:45267 <= Network.responseReceived +2ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.32", response: Object, timestamp: 253251.111828…}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 1122, encodedDataLength: 0, requestId: "4256.32", timestamp: 253251.115745}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.31", response: Object, timestamp: 253251.116596…}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 1122, encodedDataLength: 0, requestId: "4256.31", timestamp: 253251.117148}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.37", response: Object, timestamp: 253251.118293…}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 32768, encodedDataLength: 0, requestId: "4256.37", timestamp: 253251.119895}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 32768, encodedDataLength: 0, requestId: "4256.37", timestamp: 253251.120459}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 32768, encodedDataLength: 0, requestId: "4256.37", timestamp: 253251.120999}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 32768, encodedDataLength: 0, requestId: "4256.37", timestamp: 253251.121294}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 12899, encodedDataLength: 0, requestId: "4256.37", timestamp: 253251.121625}
lighthouse-background.js:45267 <= Network.loadingFinished +4ms Object {encodedDataLength: 0, requestId: "4256.37", timestamp: 253248.947116}
lighthouse-background.js:45267 <= Network.responseReceived +3ms Object {frameId: "4256.1", loaderId: "4256.3", requestId: "4256.38", response: Object, timestamp: 253251.126597…}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 32768, encodedDataLength: 0, requestId: "4256.38", timestamp: 253251.127984}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 7872, encodedDataLength: 0, requestId: "4256.38", timestamp: 253251.12903}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 0, requestId: "4256.38", timestamp: 253249.0205}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 10152, encodedDataLength: 1122, requestId: "4256.32", timestamp: 253251.12976}
lighthouse-background.js:45267 <= Network.dataReceived +2ms Object {dataLength: 0, encodedDataLength: 10152, requestId: "4256.32", timestamp: 253251.130025}
lighthouse-background.js:45267 <= Network.loadingFinished +3ms Object {encodedDataLength: 11612, requestId: "4256.32", timestamp: 253249.041206}
lighthouse-background.js:45267 <= Network.dataReceived +3ms Object {dataLength: 24209, encodedDataLength: 1122, requestId: "4256.31", timestamp: 253251.130821}
lighthouse-background.js:45267 <= Network.dataReceived +5ms Object {dataLength: 0, encodedDataLength: 24209, requestId: "4256.31", timestamp: 253251.131429}
lighthouse-background.js:45267 <= Network.loadingFinished +4ms Object {encodedDataLength: 25669, requestId: "4256.31", timestamp: 253249.129255}
lighthouse-background.js:45267 <= Log.entryAdded +280ms Object {entry: Object}
lighthouse-background.js:45267 <= Page.loadEventFired +38ms Object {timestamp: 253251.452273}
lighthouse-background.js:45267 <= Page.frameStoppedLoading +3ms Object {frameId: "4256.1"}
lighthouse-background.js:45267 <= Runtime.executionContextCreated +29ms Object {context: Object}
lighthouse-background.js:45267 Driver:warn Timed out waiting for page load. Moving on... +17s
lighthouse-background.js:45267 statusEnd Loading page & waiting for onload +1ms
lighthouse-background.js:45267 status Retrieving trace +4ms
lighthouse-background.js:45267 <= Tracing.tracingComplete +579ms Object {stream: "1"}
lighthouse-background.js:45267 status Retrieving devtoolsLog and network records +4s
lighthouse-background.js:45267 status Retrieving: URL +20ms
lighthouse-background.js:45267 status Retrieving: Viewport +2ms
lighthouse-background.js:45267 <= DOM.setChildNodes +5ms Object {nodes: Array(11), parentId: 5}
lighthouse-background.js:45267 status Retrieving: ViewportDimensions +3ms
lighthouse-background.js:45267 status Retrieving: ThemeColor +4ms
lighthouse-background.js:45267 status Retrieving: Manifest +7ms
lighthouse-background.js:45267 status Retrieving: ChromeConsoleMessages +4ms
lighthouse-background.js:45267 status Retrieving: ImageUsage +5ms
lighthouse-background.js:45267 status Retrieving: Accessibility +7ms
lighthouse-background.js:45267 status Retrieving: EventListeners +138ms
lighthouse-background.js:45267 <= Debugger.scriptParsed +89ms Object {endColumn: 5, endLine: 188, executionContextAuxData: Object, executionContextId: 14, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 0, endLine: 165, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 20226, endLine: 6, executionContextAuxData: Object, executionContextId: 15, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 101, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 193, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 2, endLine: 578, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 521, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 123, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 142, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 2, endLine: 141, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 2, endLine: 156, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 528, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 239, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 445, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 54, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 2, endLine: 117, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 2, endLine: 50, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 0, endLine: 35, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 219, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 592, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 25, endLine: 57, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 15, endLine: 15, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 2, endLine: 266, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 0, endLine: 630, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 0, endLine: 33, executionContextAuxData: Object, executionContextId: 17, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 1535, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 14225, endLine: 70, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +3ms Object {endColumn: 25238, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 24989, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 3204, endLine: 908, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 32540, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 42, endLine: 5108, executionContextAuxData: Object, executionContextId: 18, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 0, endLine: 104, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 1, endLine: 10, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 145, executionContextAuxData: Object, executionContextId: 16, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 93817, endLine: 0, executionContextAuxData: Object, executionContextId: 19, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 16037, endLine: 0, executionContextAuxData: Object, executionContextId: 19, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 5589, endLine: 0, executionContextAuxData: Object, executionContextId: 19, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 878, endLine: 0, executionContextAuxData: Object, executionContextId: 19, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 1, endLine: 17, executionContextAuxData: Object, executionContextId: 20, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 1548, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 1920, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 0, endLine: 10220, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 1, endLine: 4607, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 594, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 0, endLine: 201, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 133, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 0, endLine: 70569, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 2, endLine: 48, executionContextAuxData: Object, executionContextId: 21, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 39, endLine: 5032, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 11, endLine: 1559, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +1ms Object {endColumn: 11274, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 25331, endLine: 0, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 380, endLine: 9, executionContextAuxData: Object, executionContextId: 22, hasSourceURL: false…}
lighthouse-background.js:45267 <= Debugger.scriptParsed +2ms Object {endColumn: 12, endLine: 108, executionContextAuxData: Object, executionContextId: 13, hasSourceURL: false…}
lighthouse-background.js:45267 status Retrieving: AnchorsWithNoRelNoopener +82ms
lighthouse-background.js:45267 status Retrieving: AppCacheManifest +6ms
lighthouse-background.js:45267 status Retrieving: DOMStats +9ms
lighthouse-background.js:45267 status Retrieving: OptimizedImages +10ms
lighthouse-background.js:45267 status Retrieving: PasswordInputsWithPreventedPaste +2ms
lighthouse-background.js:45267 status Retrieving: ResponseCompression +3ms`

Mark

**Comment by Grawl** [comment]
i has this problem today

**Comment by connorjclark** [comment]
Upgrade your extension, this is really old. Otherwise, open a new issue and fill out the details.
